backend/src/main/java/com/messenger/backend/entity/ContactEntity.java

```diff
@@ -0,0 +1,16 @@
+package com.messenger.backend.entity;
+
+import javax.persistence.*;
+import java.util.Set;
+
+@Entity
+@Table(name ="tbl_contacts")
+public class ContactEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
```

Comment:
it's better to use sequences for this.
https://stackoverflow.com/questions/66461658/hibernate-generationtype-identity-vs-generationtype-sequence